### ISSUE TITLE
Improve accessibility of admin toolbar when zooming

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9820]

**Comment by laflannery** [comment]
There is current work to [update the admin toolbar module](https://www.drupal.org/project/drupal/issues/3020422). This would make the default layout vertical rather than horizontal and may resolve some of the issues we are currently seeing. It may be beneficial to wait on this ticket until the D.o work is completed.

**Comment by srancour** [comment]
@EWashb I don't think this is an issue we need to fix. As the user can choose to close the admin toolbar with the "manage" hamburger button at the top to see the content and thus doesn't need to scroll to see the content, this does meet WCAG 2.2 [SC 1.4.10](https://www.w3.org/WAI/WCAG22/Understanding/reflow.html). That along with Drupal looking at updating to a vertical only toolbar, I think this can be closed.
